cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -1393,23 +1419,76 @@ func (p *BicepProvider) loadParameters(ctx context.Context) (map[string]azure.Ar
 	return armParameters.Parameters, nil
 }
 
+type compiledBicepParamResult struct {
+	TemplateJson   string `json:"templateJson"`
+	ParametersJson string `json:"parametersJson"`
+}
+
+type compileBicepResult struct {
+	RawArmTemplate azure.RawArmTemplate
+	Template       azure.ArmTemplate
+	Parameters     azure.ArmParameters
```

Comment:
Could we add a comment to the effect of 

Only present when compiling a `.bicepparm` file.  Nil otherwise.

?